modules/library/http/src/main/java/org/geotools/http/SimpleHttpClient.java

```diff
@@ -195,6 +151,7 @@ private URLConnection openConnection(URL finalURL) throws IOException {
         return connection;
     }
 
+    @Deprecated
     public static class SimpleHTTPResponse implements HTTPResponse {
```

Comment:
Should it extend DefaultHTTPResponse? I'm guessing this is where the new DefaultHTTPResponse came from, when we do this we normally use inheritance to avoid duplication and deprecate the subclass.